EazFixer.Code/App.config

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I do not think that splitting up the code into 2 projects is needed. If certain logic has to be usable as a library, it should just be made public so others can reference the executable.